GAE/src/org/waterforpeople/mapping/app/web/CronCommanderServlet.java

```diff
@@ -212,4 +231,172 @@ private void purgeOrphanJobQueueRecords() {
             }
         }
     }
+
+    /**
+     * scans for and extracts geotags from image answers less than 1 month old
+     * Intended to be run every day
+     */
+    private void extractImageFileGeotags() {
+        Calendar deadline = Calendar.getInstance();
+        deadline.add(Calendar.MONTH, ONE_MONTH_AGO);
+        log.info("Starting scan for image answers, newer than: " + deadline.getTime());
+        QuestionAnswerStoreDao qaDao = new QuestionAnswerStoreDao();
+        Long cursor = null;
+        List<QuestionAnswerStore> dfjqList;
+        int retirees = 0;
+        int json = 0;
+        int nonjson = 0;
+        Media media;
+
+        do {
+            dfjqList = qaDao.listByTypeAndDate("IMAGE", cursor, deadline.getTime(), null, 1000);
```

Comment:
What is the cursor for? I checked the method and the second param is surveyId, cursor is the second before last

Reply:
True. Looked at the javadoc list...